api/src/pcapi/routes/pro/users.py

```diff
@@ -303,7 +303,9 @@ def anonymize() -> None:
         raise ResourceNotFoundError(errors={"global": "Cette fonctionnalité n'est pas disponible"})
 
     user = current_user._get_current_object()
-    anonymized = anonymize_pro_user(user)
+    anonymized = anonymize_pro_user(
+        user=user, action_history_comment="Demande d’anonymisation depuis l’espace partenaire"
+    )
```

Comment:
```suggestion
    anonymized = anonymize_pro_user(user=user, author=user)
```
me semble plus pérenne qu'enregistrer une chaîne de caractère dans la base de donnée

sauf s'il y a plusieurs points d'entrée pour que le pro demande à être anonymisé, le backoffice pourra faire le test `action_history.author == action_history.user` pour savoir quoi afficher